system-test/maxtest/include/maxtest/envv.hh

```diff
@@ -8,26 +8,26 @@
  *
  * @param name Name of the variable
  * @param format Default value format string
- * @return Enviromental variable value
+ * @return Environment variable value
  */
 std::string readenv(const char* name, const char* format, ...) __attribute__ ((format (printf, 2, 3)));
 
 std::string envvar_get_set(const char* name, const char* format, ...)
 __attribute__ ((format (printf, 2, 3)));;
 
 /**
- * @brief readenv_int Read integer value of enviromental variable, if empty - set dafault
+ * @brief readenv_int Read integer value of environmental variable, if empty - set default
  * @param name Name of the variable
  * @param def Default value
- * @return Enviromental variable value converted to int
+ * @return Environment variable value converted to int
```

Comment:
I only realized that the capital item probably didn't need the `-al` suffix and manually fixed it. I think the lowercase item should be changed as well...